### DISCUSSION TITLE
The use of weather data

### DISCUSSION THREAD

**Discussion by HaroleDev** [open-meteo/open-meteo#134] [Q&A]
I'm not referring to the Open-Meteo API itself, but rather to the weather data the API is requesting. The README file stated under the [Data License](https://github.com/open-meteo/open-meteo#data-license) part as CC-BY-NC, and there isn't an exception for when to use it commercially. Maybe it's just a part of my misunderstanding reading it.

**Comment by KlemenSpruk** [comment]
Is this valid for your API only? If we deploy our own instance, we can use api call result also for commercial purposes as I understand. Or am I wrong?

Thanks, Klemen

**Comment by patrick-zippenfenig** [comment]
Hey,
sorry for the late reply. It is correct, that data is currently offered as CC-BY-NC and therefore commercial use of data is not allowed.

However, I checked with all upstream licences and it is possible to switch to CC-BY and allow commercial use of data. In terms of API service, it will stay free for non-commercial use, but the data licence will switch to CC-BY. At some point a SaaS for commercial use will follow with proper SLAs.

Running your own instance is possible as well. Please be aware that there is no support and the code has not yet reached a stable version. Within the next weeks there will be breaking changes from data from national weather services (e.g. the German weather service is changing resolution on the 23. November) and I am also working on a cluster oriented deployment, because a single server has issues to keep up with the amount of data to process

Best, Patrick

**Reply by KlemenSpruk** [reply]
Thanks Patrick!
